packages/typescript/src/components/Interface.tsx

```diff
@@ -116,6 +116,7 @@ export function InterfaceMember(props: InterfaceMemberProps) {
           kind="type"
           nameKind="interface-member"
           flags={OutputSymbolFlags.StaticMember}
+          nullish={props.optional}
```

Comment:
I assume the use case here is its up to the user to know that the type can have `null` as a variant and so we pass this to be explicit?

Reply:
Yeah, and thinking on this, maybe we should add 'nullish' prop to InterfaceMemberProps as well